docs/Getting Started/Debian/Debian Trixie Root on ZFS.rst

```diff
@@ -0,0 +1,1207 @@
+
+   If you do nothing extra, ``/tmp`` will be stored as part of the root
+   filesystem. Alternatively, you can create a separate dataset for ``/tmp``,
+   as shown above. This keeps the ``/tmp`` data out of snapshots of your root
+   filesystem. It also allows you to set a quota on ``rpool/tmp``, if you want
+   to limit the maximum space used. Otherwise, you can use a tmpfs (RAM
+   filesystem) later.
+
+   **Note:** If you separate a directory required for booting (e.g. ``/etc``)
+   into its own dataset, you must add it to
+   ``ZFS_INITRD_ADDITIONAL_DATASETS`` in ``/etc/default/zfs``.  Datasets
+   with ``canmount=off`` (like ``rpool/usr`` above) do not matter for this.
+
+#. Mount a tmpfs at /run::
+
+     mkdir /mnt/run
+     mount -t tmpfs tmpfs /mnt/run
+     mkdir /mnt/run/lock
+
+#. Install the minimal system::
+
+     debootstrap trixie /mnt
+
+   The ``debootstrap`` command leaves the new system in an unconfigured state.
+   An alternative to using ``debootstrap`` is to copy the entirety of a
+   working system into the new ZFS root.
+
+#. Copy in zpool.cache::
+
+     mkdir /mnt/etc/zfs
+     cp /etc/zfs/zpool.cache /mnt/etc/zfs/
+
+Step 4: System Configuration
+----------------------------
+
+#. Configure the hostname:
+
+   Replace ``HOSTNAME`` with the desired hostname::
+
+     hostname HOSTNAME
+     hostname > /mnt/etc/hostname
+     vi /mnt/etc/hosts
+
+   .. code-block:: text
+
+     Add a line:
+     127.0.1.1       HOSTNAME
+     or if the system has a real name in DNS:
+     127.0.1.1       FQDN HOSTNAME
+
+   **Hint:** Use ``nano`` if you find ``vi`` confusing.
+
+#. Configure the network interface:
+
+   Find the interface name::
+
+     ip addr show
+
+   Adjust ``NAME`` below to match your interface name::
+
+     vi /mnt/etc/network/interfaces.d/NAME
+
+   .. code-block:: text
+
+     auto NAME
+     iface NAME inet dhcp
+
+   Customize this file if the system is not a DHCP client.
+
+#. Optional: Install driver firmware and WiFi support
+
+   If you're installing on a laptop or a device where wireless is the
+   primary network option, the above may not be sufficient as you
+   could lack the appropriate firmware for the device and tools to
+   configure the radio. Install some additional packages to cover
+   that need::
+
+     apt install --yes firmware-linux wireless-tools
+
+#. Configure the package sources::
+
+     vi /mnt/etc/apt/sources.list
+
+   .. code-block:: sourceslist
+
+     deb http://deb.debian.org/debian trixie main contrib non-free-firmware
+     deb-src http://deb.debian.org/debian trixie main contrib non-free-firmware
+
+     deb http://deb.debian.org/debian-security trixie-security main contrib non-free-firmware
+     deb-src http://deb.debian.org/debian-security trixie-security main contrib non-free-firmware
+
+     deb http://deb.debian.org/debian trixie-updates main contrib non-free-firmware
+     deb-src http://deb.debian.org/debian trixie-updates main contrib non-free-firmware
```

Comment:
Debian Trixie is using:

/etc/apt/sources.list.d/debian.sources

https://wiki.debian.org/SourcesList

Types: deb deb-src
URIs: https://deb.debian.org/debian
Suites: trixie trixie-updates
Components: main contrib non-free-firmware
Signed-By: /usr/share/keyrings/debian-archive-keyring.gpg

Types: deb deb-src
URIs: https://security.debian.org/debian-security
Suites: trixie-security
Components: main contrib non-free-firmware
Signed-By: /usr/share/keyrings/debian-archive-keyring.gpg

Reply:
I installed a new system with trixie and those will are create. So everything is fine. :-D